components/tools/samlsso-validator/org.wso2.carbon.identity.tools.saml.validator.ui/src/main/resources/web/tools-saml/saml_request_validate.jsp

```diff
@@ -141,7 +142,7 @@
 							Request <span class="required">*</span></td>
 						<td style="height: 200px;"><textarea type="text"
 								name="samlReqest" id="samlReqest" class="text-box-big"
-								style="width: 50%; min-height: 180px;" autocomplete="off"><%=(isDoValidate && samlRequest != null) ? samlRequest : ""%></textarea>
+								style="width: 50%; min-height: 180px;" autocomplete="off"><%=(isDoValidate && samlRequest != null) ? Encode.forHtml(samlRequest): ""%></textarea>
```

Comment:
Since this is a text area there's no need to use Encode.forHtml. Use Encode.forHtmlContent. Further seems you have not removed previous encoding of samlrequest (https://github.com/wso2/carbon-identity/pull/998/files#diff-9291b12396112fd41fafe91ac0dd0f59R95). Thus, this encodes samlrequest twice.